release/pkg/manager/calico/manager.go

```diff
@@ -101,12 +104,18 @@ type ReleaseBuilder struct {
 	// validate is a flag to indicate that we should skip pre-release validation.
 	validate bool
 
+	// validateBranch is a flag to indicate that we should skip release branch validation.
+	validateBranch bool
+
 	// calicoVersion is the version of calico to release.
 	calicoVersion string
 
 	// operatorVersion is the version of the operator to release.
 	operatorVersion string
 
+	// chartVersion is the version of the helm chart to release.
```

Comment:
We should at least comment that this field is defunct and not used. Personally I'd probably just not add it to OSS since we don't need it here, though.